data/avro-schema/src/main/resources/avro/net/corda/data/membership/db/response/query/MemberSignature.avsc

```diff
@@ -0,0 +1,18 @@
+{
+  "type": "record",
+  "name": "MemberSignature",
```

Comment:
We may run in to issues here during reregistration where a member changes their member context and signature. They may not be approved immediately so we might need multiple records per member. Some of the details still need to be figured out.

Reply:
Would we ever want to use historical member signatures?

Reply:
I'm just thinking of the scenario down the line with reregistration when alice could submit a new member context which the MGM needs to approve. But before it's approved, the member list is sent to bob. If we persist the member signature for the new member context it will not match the approved version of alice's member context. We will have a similar issue for the member info though and how to persist active and pending versions of a member info if we choose to allow both states at once. This is probably something that we will just need to think about when we get to reregistration.